README.md

```diff
@@ -21,14 +21,19 @@ NOTE: CARAT was developed for crystallographic groups in dimensions up to 6.
 Most algorithms also work in higher dimensions. However, integer overflow
 is not trapped in general.
 
+## Website
+
+You can find the official CARAT homepage and documentation on:
+[https://lbfm-rwth.github.io/carat/](https://lbfm-rwth.github.io/carat/)
 
 ## Installation
 
 The easiest way to compile CARAT is to go to the CARAT base directory
-(the one containing this REAMDME), and issue the following
+(the one containing this README.md), and issue the following commands:
+     ./autogen.sh (only if you are building CARAT directly from GitHub)
+     ./configure
+     make
 
-    ./configure
-    make
```

Comment:
You removed the empty line before the code snippet. That causes it to misrender (in Markdown, it will be treated as part of the paragraph before it, no matter how much it is indented. That's a feature, not a bug). Please add it back.